abci/preblock/oracle/preblock_test.go

```
@@ -402,6 +402,31 @@
 			require.Equal(s.T(), uint64(1), nonce)
 		}
+
+		event := s.ctx.EventManager().Events()[len(s.ctx.EventManager().Events())-1]
+		s.Require().Equal("oracle_prices", event.Type)
+		s.Require().Equal(3, len(event.Attributes))
+
+		// check the event matches the expected event
+		expectedPriceAttrs := []sdk.Attribute{}
+		for idx, cp := range s.currencyPairs {
+			var bigIntPrice big.Int
+			if err := bigIntPrice.GobDecode(prices[uint64(idx)]); err != nil {
```

golangci-lint (1.62.2)

[high] G115: integer overflow conversion int -> uint64

(gosec)